flyingpigeon/processes/wps_pointinspection.py

```diff
@@ -0,0 +1,131 @@
+            #               ),
+        ]
+
+        super(PointinspectionProcess, self).__init__(
+            self._handler,
+            identifier="pointinspection",
+            title="Point Inspection",
+            abstract='Extract the timeseries at the given coordinates.',
+            version="0.10",
+            metadata=[
+                Metadata('LSCE', 'http://www.lsce.ipsl.fr/en/index.php'),
+                Metadata('Doc', 'http://flyingpigeon.readthedocs.io/en/latest/'),
+            ],
+            inputs=inputs,
+            outputs=outputs,
+            status_supported=True,
+            store_supported=True,
+        )
+
+    def _handler(self, request, response):
+        # init_process_logger('log.txt')
+        # response.outputs['output_log'].file = 'log.txt'
+
+        ncs = extract_archive(
+            resources=rename_complexinputs(request.inputs['resource']))
+        LOGGER.info('ncs: {}'.format(ncs))
+
+        coords = []
+        for coord in request.inputs['coords']:
+            coords.append(coord.data)
+
+        LOGGER.info('coords {}'.format(coords))
+        filenames = []
+        nc_exp = sort_by_filename(ncs, historical_concatination=True)
+
+        for key in nc_exp.keys():
+            try:
+                LOGGER.info('start calculation for {}'.format(key))
+                ncs = nc_exp[key]
+                times = get_time(ncs)
+                concat_vals = times
+                header = 'date_time'
+                filename = '{}.csv'.format(key)
+                filenames.append(filename)
+
+                for p in coords:
+                    try:
+                        response.update_status('processing point: {}'.format(p), 20)
+                        # define the point:
+                        p = p.split(',')
+                        point = Point(float(p[0]), float(p[1]))
+
+                        # get the values
+                        timeseries = call(resource=ncs, geom=point, select_nearest=True)
+                        vals = get_values(timeseries)
+
+                        # concatenation of values
+                        header = header + ',{}-{}'.format(p[0], p[1])
+                        concat_vals = column_stack([concat_vals, vals])
+                    except Exception as e:
+                        LOGGER.debug('failed for point {} {}'.format(p, e))
+                response.update_status('*** all points processed for {0} ****'.format(key), 50)
+
+                # TODO: Ascertain whether this 'savetxt' is a valid command without string formatting argument: '%s'
+                savetxt(filename, concat_vals, fmt='%s', delimiter=',', header=header)
+            except Exception as ex:
+                LOGGER.debug('failed for {}: {}'.format(key, str(ex)))
+
+        # set the outputs
+        response.update_status('*** creating output tar archive ****', 90)
+        tarout_file = archive(filenames)
```

Comment:
why not use `output_dir=self.workdir`?

Reply:
Done: 
`tarout_file = archive(filenames, output_dir=self.workdir)`